charts/base-cluster/templates/ingress/traefik.yaml

```diff
@@ -115,8 +115,6 @@ spec:
         preferServerCipherSuites: true
         minVersion: VersionTLS12
 
-    podSecurityPolicy:
-      enabled: true
```

Comment:
we can template it for older k8s version if required

Reply:
If someone is at that k8s version, an update is overdue, right?